schemas/context/experienceevent.schema.json

```diff
@@ -126,6 +126,31 @@
           "description": "The media activity information related to the experience event"
         }
       },
+      "xdm:profileStitching" : {
+        "title": "Profile Stitching",
+        "description": "Details about the ids that were joined by profile stitching.",
+        "$ref": "https://ns.adobe.com/xdm/context/profileStitch"
+      },
+      "xdm:segmentMemberships": {
```

Comment:
What is a "segment", where it is defined, is it specific to an Adobe workflow or generic?

Reply:
It's not defined yet in it's own xdm. This pull request didn't introduce the concept of segment membership, it already exists on profile and the PR for the segment membership is merged. I'm just adding it experience event. It's meant to be generic, but this will definitely be used by Audience Manager and I think Target.

Reply:
Got it.   OK - no issue there.